src/app/features/genes/components/gene-soe-charts/gene-soe-charts.component.html

```diff
@@ -1,20 +1,22 @@
 <div *ngIf="_gene" class="gene-score-charts">
-  <div *ngFor="let chart of charts">
+  <div *ngFor="let chart of scoreDistributions; let i = index">
```

Comment:
Don't seem to be using `i` here, could you remove it?
```suggestion
  <div *ngFor="let chart of scoreDistributions">
```

Reply:
deleted!